dev-docs/decision-records/unstable-plugins.md

```diff
@@ -23,14 +23,13 @@ Part of the motivation for this decision, was the recognition that we already ha
 
```

Comment:
The changes to this document don't seem consistent with the changes to the code. For instance, the code has a `PluginPrivate`, but it's removed from the DR. Are these changes not properly synced up?

Reply:
Suspect this file just needs to be reverted

Reply:
it's reverted now